topdown/http.go

```diff
@@ -139,12 +139,22 @@ func addHeaders(req *http.Request, headers map[string]interface{}) (bool, error)
 	for k, v := range headers {
 		// Type assertion
 		header, ok := v.(string)
-		if ok {
+		if !ok {
+			return false, fmt.Errorf("invalid type %T for headers value", v)
```

Comment:
🤔 I'm torn. It's nice to know the wrong type, but it's also leaking implementation details: 
```
> http.send({ "method": "get", "url": "http://127.0.0.1:8181", "headers": { "Host": ["foo"] }})
1 error occurred: http.send({ "method": "get", "url": "http://127.0.0.1:8181", "headers": { "Host": ["foo"] }}): eval_builtin_error: http.send: invalid type []interface {} for headers value
```
In the message, the type `[]interface{}` has little to do with Rego, it's the Golang type. I think this could be confusing. It would probably be better to properly check the passed headers, but that would be out of scope for this PR...

As a compromise for now, what do you think of this:
```go
return false, fmt.Errorf("invalid type for headers value %q", v)
```
That would yield the following error,
```
1 error occurred: http.send({ "method": "get", "url": "http://127.0.0.1:8181", "headers": { "Host": ["foo"] }}): eval_builtin_error: http.send: invalid type for headers value ["foo"]
```
which is more helpful than the current one, but also doesn't show golang types... 💭

Reply:
Yep, I agree that using `%q` will give a much better message.

Reply:
Fixed.